src/index.css

```diff
@@ -87,3 +85,7 @@ td {
 .tracker-table-foot {
   background-color: #d4dff9;
 }
+
+.download-button {
+  background-color: rgb(20, 130, 73);
```

Comment:
why do you use different type of colors?
sometimes it is #d4dff9, and sometimes it is rgb(20, 130, 73) ?

Reply:
Done. Thank you.